app/controllers/anonymous_feedback_controller.rb

```diff
@@ -6,27 +6,38 @@ class AnonymousFeedbackController < RequestsController
   def index
     authorize! :read, :anonymous_feedback
 
-    if index_params[:path].present?
-      api_response = fetch_anonymous_feedback_from_support_api
-
-      if api_response["pages"] > 0 && api_response["current_page"] > api_response["pages"]
-        # assume user has gone past last page, redirect to first page
-        redirect_to anonymous_feedback_index_path(index_params.merge(page: 1))
-      else
-        @feedback = AnonymousFeedbackPresenter.new(api_response)
-        @from_date = Date.parse(api_response["from_date"]).to_s(:govuk_date_short) if api_response["from_date"]
-        @to_date = Date.parse(api_response["to_date"]).to_s(:govuk_date_short) if api_response["to_date"]
-
-        respond_to do |format|
-          format.html
-          format.json { render json: @feedback.to_json }
-        end
-      end
-    else
+    unless has_required_api_params?
       respond_to do |format|
         format.html { redirect_to anonymous_feedback_explore_url, status: 301 }
-        format.json { render json: {"errors" => ["Please set a valid 'path' parameter"] }, status: 400 }
+        format.json { render json: {"errors" => ["Please set a valid 'path' or 'organisation' parameter"] }, status: 400 }
       end
+      return
+    end
+
+    api_response = fetch_anonymous_feedback_from_support_api
+
+    if api_response.beyond_last_page?
+      redirect_to first_page
+      return
+    end
+
+    respond_to do |format|
+      format.html {
+        @feedback = AnonymousFeedbackPresenter.new(api_response)
+        @dates = present_date_filters(api_response)
+        # TODO: we should decide how to describe filtering by both a path and
+        # an organisation, separately and together
+        # TODO: I guess we should determine this filtering information from the
+        # api_response rather than user-supplied params
+        # TODO: this shouldn't be here, it belongs in its own object, possibly
+        # combined with date filters in a more general filters presenter?
+        if index_params[:path].present?
+          @filtered_by = index_params[:path]
```

Comment:
Would it be nicer if we were working with the `path` and/or `organisation` from the API response instead of user-supplied params here? (As described in that TODO just above.)